src/payment/models/payout/strategies/prepare/impl/ethereum.strategy.ts

```diff
@@ -1,10 +1,17 @@
 import { Injectable } from '@nestjs/common';
+import { Blockchain } from 'src/blockchain/shared/enums/blockchain.enum';
+import { Asset, AssetType } from 'src/shared/models/asset/asset.entity';
+import { AssetService } from 'src/shared/models/asset/asset.service';
 import { PayoutOrderRepository } from '../../../repositories/payout-order.repository';
 import { EvmStrategy } from './base/evm.strategy';
 
 @Injectable()
 export class EthereumStrategy extends EvmStrategy {
-  constructor(payoutOrderRepo: PayoutOrderRepository) {
+  constructor(protected readonly assetService: AssetService, payoutOrderRepo: PayoutOrderRepository) {
     super(payoutOrderRepo);
   }
+
+  protected getFeeAsset(): Promise<Asset> {
+    return this.assetService.getAssetByQuery({ dexName: 'ETH', blockchain: Blockchain.ETHEREUM, type: AssetType.COIN });
```

Comment:
Should we put the fee asset descriptors in a central place? It felt like I saw them 100 times 😆

Reply:
understand your point, but as discussed its not a big deal - lets keep it since its basically just a config per strategy